apps/omg/test/support/test_helper.ex

```diff
@@ -150,17 +150,22 @@ defmodule OMG.TestHelper do
   def write_fee_file(map, file_name) when is_map(map) do
     {:ok, json} =
       map
-      |> Enum.map(fn {"0x" <> _ = token, fee} ->
-        %{
-          token: token,
-          amount: fee.amount,
-          subunit_to_unit: fee.subunit_to_unit,
-          pegged_amount: fee.pegged_amount,
-          pegged_currency: fee.pegged_currency,
-          pegged_subunit_to_unit: fee.pegged_subunit_to_unit,
-          updated_at: fee.updated_at
-        }
+      |> Enum.map(fn {tx_type, fees} ->
```

Comment:
lets try simplifying this and reduce the amount of anonymous functions in pipelines